config/routes.rb

```diff
@@ -5,7 +5,11 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   resources :foods, except: [:update]
-  resources :inventories, only: [:index, :show, :destroy, :new, :create]
+  resources :inventories, only: [:index, :show, :destroy, :new, :create] do
+    get 'foods/new', to: 'inventory_foods#new'
+    post 'foods', to: 'inventory_foods#create'
+    delete 'foods/:id', to: 'inventory_foods#destroy', as: 'food'
```

Comment:
You can specify using resources instead of writing individual paths
`resources :inventory_foods, path: 'food', only: [:destroy, :new, :create]`

Reply:
I tried to do this but I get routes like these ones.
```
Prefix                       Verb       URI Pattern                                         Controller#Action
inventory_inventory_foods    POST       /inventories/:inventory_id/food(.:format)           inventory_foods#create
new_inventory_inventory_food GET        /inventories/:inventory_id/food/new(.:format)       inventory_foods#new
inventory_inventory_food     DELETE     /inventories/:inventory_id/food/:id(.:format)       inventory_foods#destroy
```
Having 'inventory' repeated didn't seem right. I'm still open to change this if you thing the prefixes are okay.